content/rfc/0002-personal-access-tokens-management.adoc

```diff
@@ -0,0 +1,261 @@
+== 2. Personal Access Tokens Management and Authentication
+
+[width="100%",cols="<18%,<82%",]
+|===
+|Feature Name | Personal Access Tokens Management and Authentication
+|Start Date | Aug26th, 2025
+|Category | Architecture, Authnz
+|PR | https://github.com/trento-project/docs/pull/92[#92]
+|===
+
+:toc:
+
+== Summary
+
+Add Personal Access Tokens (PATs) management and authentication.
+
+== Motivation
+
+We want to simplify third-party clients integrations with Trento's APIs.
+
+Currently such clients would have to:
+
+ * perform a login API request with username/password
+ * use the provided `access_token` to make subsequent API calls
+ * refresh the token by using the `refresh_token` provided at login
+ * use the *new* `access_token` for subsequent API calls
+
+While this approach works well for user-interactive scenarios as the web UI, it may not be ideal for programmatic access or third-party integrations because:
+
+ * to perform initial login, a username/password pair needs to be known/stored by the client application, which may not be feasible or secure in all cases
+ * the `access_token` issued at login is short-lived, with a default lifetime of 3 minutes, unless a global configuration is set to extend it
+ * the refresh token flow adds complexity to the client requiring it to track current access token expiration or react to unauthorized requests to trigger refresh
```

Comment:
Not only is complexity the problem; right now, we can't perform the token retrieval/fetch without _being Trento (Web)_, as the `redirect_uri` is not configurable.